documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,465 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Migration from OpenStack moves to being a fully supported feature
+
+In this version of {project-short}, migration using {osp} source providers graduated from a Technology Preview feature to a fully supported feature.
+
+.Integration of the create and update provider user interface
+
+The user interface of the source providers and destination providers now aligns with the look and feel of the {ocp} web console and displays the current data.
+
+.Standalone UI
+
+The old UI of {project-short} 2.3 cannot run by setting `feature_ui: true` in ForkliftController anymore.
+
+.Improved errors logged in populator pods
+
+In earlier releases of {project-short} {project-version}, populator pods were always restarted on failure. This made it difficult to gather the logs from the failed pods. {project-short} 2.5.3 has a maximum number of three restarts of populator pods. On the third and final time, the populator pod remains in the fail status and its logs can then be easily gathered by must-gather and by forklift-controller to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
+
+.The `VirtualMachineClusterFlavor` CRDs replace {ocp-name} templates
+
+The Virtual Machine Flavor API replaces {ocp-name} templates. {project-short} currently falls back to using {ocp-name} templates when a relevant preference is not available.
+
+.Full support for migration from OVA
+
+{project-short} {project-version} supports migration from OVA.
+
+.{ocp-name} Console `Create/Edit` pages replace Forms and Wizards
+
+Forms and Wizards that use `@migtools/lib-ui` do not function as expected when running inside the plugin. The migtool library was built as a standalone application. The `Create` and `Edit` pages in the {ocp-name} Console replace the Forms and Wizards. link:https://issues.redhat.com/browse/MTV-615[(MTV-615)]
+
+.Scan OVA files in the root folder of an NFS share
+
+Migration scans the files directly to the NFS share folder, not to a separate folder. link:https://issues.redhat.com/browse/MTV-680[(MTV-680)]
+
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
```

Comment:
I do not understand why you are linking to Jira tickets for new features

Would links to documentation not be better? Reader looks at the brief overview of the new feature and then clicks on to the docs for the new feature.

Also read Richard's tickets and borrow text from those docs to fill out this section.


Furthermore, you are looking at this section as if it will only be used for 2.6.0... what happens if Arik wants to include a great new feature in MTV 2.61... how will the reader know this feature is for 2.6.1 and which other features were earlier.

This section does not read particularly well. Ask Richard and Arik for their top five to ten new features, add these. Then add links to the docs (use hard links.... as the links will not change for the 2.6 version too much especially if you go to the top level of the doc. 

@ahadas and @RichardHoch - please can I ask for your input on the New Features and ask for a list of five to ten new features that will really grab the attention of the user? 

@HagayVider1 - do not do a single line for these big new features, write a short paragrapgh. Richard's docs tend to have good introductions on the benefits, which will allow you to almost copy and paste.    

Document these features using a standard introductory text, such as "In MTV 2.6.0, you can ...." 

@ahadas and @RichardHoch - WDYT - worth it? 

Once you have these top level features, move this section to a `Level 3 ===` heading and call it like **2.6.0 new features summary**

Reply:
But what happens you release 2.6.1.

As we group release notes on the Y not Z release, i think it might be better to be safe than sorry

Reply:
@anarnold97 My 2 cents -- MTV does a pretty good job of introducing features only in the y-stream releases, that is the y.0 releases, so I'd be fine with omitting the .0 in 2.6.0 for this section. For known and resolved issues, specifying the z-stream is necessary. On the other hand, not all projects in the MMS/OADP universe follow the "no new features in z-stream" rule, so I can see using  z-stream numbers across the board in the new features section of MMS/OADP release notes.